assets/tests/components/detours/diversionPage.test.tsx

```diff
@@ -129,30 +263,90 @@ describe("DiversionPage", () => {
     expect(screen.getByTitle("Detour End")).not.toBeNull()
   })
 
-  test("when end point has been set, finish detour button is visible", async () => {
+  test("clicking on route shape after the detour is ended doesn't do anything", async () => {
     const { container } = render(<DiversionPage />)
 
     fireEvent.click(originalRouteShape.get(container))
 
-    expect(
-      screen.getByRole("button", { name: "Finish Detour", hidden: true })
-    ).not.toBeVisible()
+    fireEvent.click(container.querySelector(".c-vehicle-map")!)
 
     fireEvent.click(originalRouteShape.get(container))
 
-    expect(screen.getByRole("button", { name: "Finish Detour" })).toBeVisible()
+    await waitFor(() => {
+      expect(screen.getByTitle("Detour End")).not.toBeNull()
```

Comment:
https://github.com/mbta/skate/pull/2580#discussion_r1603769317

Reply:
[Feedback: Replace instances of not.toBeNull() with toBeVisible()](https://github.com/mbta/skate/pull/2580/commits/e9ce55cc8a39b533b7fac7d761f6c942c1651bc8)

Reply:
> Maybe a copy-pasta from somewhere?
It certainly seems to be prevalent in this file from before these changes